lib/routes/create_invoice/create_invoice_page.dart

```diff
@@ -63,6 +68,20 @@ class CreateInvoicePageState extends State<CreateInvoicePage> {
         }
       },
     );
+    _fetchLightningLimits();
+  }
+
+  void didChangeAppLifecycleState(AppLifecycleState state) {
+    if (state == AppLifecycleState.resumed) {
+      _fetchLightningLimits();
```

Comment:
I think it is better to handle these refreshes automatically from within the bloc.
In c-breez we use FGBGEvents at the bloc level to refresh data on resume.

Reply:
Also perhaps we can instantiate the LNurlCubit only around the pages uses it? I don't know how much effort needed to achieve it though.

Reply:
> Also perhaps we can instantiate the LNurlCubit only around the pages uses it? I don't know how much effort needed to achieve it though.

It's achievable but let's address these in scope of another PR. I'm creating the issues.

Reply:
https://github.com/breez/l-breez/issues/67